README.md

```diff
@@ -437,17 +494,17 @@ jobs:
 
 1. The regular expression in step **Check tag** should be:
 
-```
+```regexp
 ^refs/tags/v[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}$
 ```
 
-2. The output format in step **Get version** should be:
+1. The output format in step **Get version** should be:
```

Comment:
Not fully sure if this works correctly for this case because there are multiple lines. But the linter said it's to change like this. Same below.

Reply:
Does not work, will revert this change...